example/profiles/default.profile

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
```

Comment:
I think `example` is a better ID than `default`.
The `default` profile is implicit to be the Benchmark "as is", with the rule selection not being "altered" by a profile.
Also, it can be less misleading to us when somebody says they tried the `example` profile, rather then the `default` profile.

Reply:
Okay, I changed this to `example` profile in `example` product.